lohica-js-network/src/components/inputComponents/SurnameInput.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {FormControl, FormGroup, ControlLabel, Col, Alert} from 'react-bootstrap';
+import {connect} from 'react-redux';
+import { validateSurname } from '../../actions/inputAction'
+import {nameMidSurRegExp} from './handlers/regExpInputValid';
```

Comment:
is it regex for name middle name and surname?

Reply:
named nicely